Patch to document esp-udp algo for setkey as well as lack of tcp-md5 support for
linux systems

--- ipsec-tools-0.7.3.orig/src/setkey/setkey.8	2010-03-23 13:02:39.000000000 +0100
+++ ipsec-tools-0.7.3/src/setkey/setkey.8	2010-03-23 13:03:52.000000000 +0100
@@ -293,6 +293,8 @@
 ESP based on rfc2406
 .It Li esp-old
 ESP based on rfc1827
+.It Li esp-udp
+ESP-UDP based on rfc3948
 .It Li ah
 AH based on rfc2402
 .It Li ah-old
@@ -735,7 +737,7 @@
 				ah-old: 128bit ICV (no document)
 aes-xcbc-mac	128		ah: 96bit ICV (RFC3566)
 		128		ah-old: 128bit ICV (no document)
-tcp-md5		8 to 640	tcp: rfc2385
+tcp-md5		8 to 640	tcp: rfc2385 (tcp-md5 support only on BSD)
 .Ed
 .Pp
 These encryption algorithms can be used as
